test_util/integration_test.py

```diff
@@ -1260,12 +1260,19 @@ def wait_for_endpoint():
 
     wait_for_endpoint()
 
-    test_cache_url = "http://{}:{}/signal_test_cache".format(service_points[0].host, service_points[0].port)
+    test_host = service_points[0].host
+    test_port = service_points[0].port
+    test_cache_url = "http://{}:{}/signal_test_cache".format(test_host, test_port)
+
     cmd = """
-data=`/opt/mesosphere/bin/dcos-signal -report-host leader.mesos -test 2> /dev/null`;
-curl -H 'Content-Type: application/json' -X POST -d "$data" {};
+ID_PATH="/${PWD}/test-cluster-id"
+echo -n 'test-id' > $ID_PATH
```

Comment:
the dcos-cluster-id service actually puts a new line into the cluster-id file, the signal service should do a `.strip()` to remove whitespace around the cluster id